cluster/examples/kubernetes/ceph/flex/storageclass.yaml

```diff
@@ -18,6 +18,7 @@ kind: StorageClass
 metadata:
   name: rook-ceph-block
 provisioner: ceph.rook.io/block
+allowVolumeExpansion: true
```

Comment:
It's probably best to add a `works for Kubernetes 1.14+` note as a reminder to all users.